src/components/pagination/index.js

```diff
@@ -42,7 +43,7 @@ const Pagination = props => {
             className="pagination_select"
             value={props.rowsPerPage}
             options={pageOptions}
-            onChange={props.onSelectRowsPerPage}
+            onChange={() => console.log('onchsnge wsjbdefej') || props.onSelectRowsPerPage()}
```

Comment:
whats this?

Reply:
@SomashekarGowdaS dude check this please